robmuxinator/robmuxinator.py

```diff
@@ -763,6 +763,12 @@ def main():
         help="the path to the yaml config file",
         default="/etc/ros/upstart_robot.yaml",
     )
+    parser.add_argument(
+        "-n",
+        "--number",
+        help="the number to append to each session name within the config file. used for multi robot setups. Can also be used within sessions as a env_variable. this is used as a unique ros domain id",
+        default="",
+    )
```

Comment:
I'm a bit hesitant to just call this `number`
Maybe we should rather call this `session_id_no` or `session_prefix_no` or something

We should maybe also parse this argument as an integer explicitly to have some type checking

It is later used as `ROS_DOMAIN_ID` - is this really guaranteed to be unique?

Reply:
https://github.com/4am-robotics/orga/wiki/ROS-Domain-Ids starts with `7`
I'll add `reserved` for `0-6` so no one is going to assign those `ROS_DOMAIN_ID`s

Reply:
or maybe uuid?

Reply:
I would lean towards session_prefix_no

Reply:
`instance_id` it is - as discussed with @Odin-byte